test/extended/operators/cluster.go

```diff
@@ -18,7 +18,7 @@ import (
 var _ = g.Describe("[Feature:Platform] Managed cluster should", func() {
 	defer g.GinkgoRecover()
 
-	g.It("have no crashlooping pods in core namespaces over two minutes", func() {
+	g.It("have no crashlooping pods in core namespaces over two minutes [IgnoreInDR]", func() {
```

Comment:
Why are things crash looping after DR?

Reply:
At least openshift-sdn and openshift-apiserver are still running and attempting to reach kube apiserver

Reply:
How long does it take to clear?

Reply:
It won't clear - this test would fail if the pod has restarted more than 5 times. In order to clear that the pod should be removed - and we'll lose its logs

Reply:
No, we would just add a condition to the test that makes this test tolerant of clusters that have been running for a while.